app/lib/actions/katello/product/content_create.rb

```diff
@@ -19,8 +19,6 @@ def plan(repository)
                                     product_id: repository.product.cp_id,
                                     content_id: content_id)
 
-            else
-              content_id = repository.content_id
```

Comment:
We might still need this assignment of content_id - because it looks like it may be referenced a few lines down. Quick take on that before I test it?

Reply:
Good catch.  I'll update it to not set it if its already set, as I think that makes more sense than setting if its already there.  Def. test custom repo creation and redhat repo enablement.

Reply:
actually, i see its probably simpler just to leave this else block